### PR TITLE
Autosize with markdown

### PR DESCRIPTION
**Pull request by harish86**

### CODE REVIEW

File: web/static/components/contacts/to-do-edit.vue [hunk 1]
```diff
@@ -116,8 +116,8 @@
     components: {
       'inline-edit': InlineEdit,
       'checkbox': VueStrap.checkbox,
-      'el-date-picker': Element.DatePicker,
-      'el-checkbox': Element.Checkbox
+      'el-date-picker': ElementUi.DatePicker,
+      'el-checkbox': ElementUi.Checkbox
```
Comment (parallel588) [inline web/static/components/contacts/to-do-edit.vue]: ElementUI is undefined. I think it's typo. ElementUI -> Element @antoineherzog @harish86

Comment (parallel588) [inline web/static/components/contacts/to-do-edit.vue]: 119:25  error  'ElementUi' is not defined  no-undef
  120:22  error  'ElementUi' is not defined  no-undef

Comment (harish86) [inline web/static/components/contacts/to-do-edit.vue]: @parallel588 it is updated in brunch-config as well. 'Element' had a conflict with the autosize library, I have renamed it to ElementUi.

Comment (parallel588) [inline web/static/components/contacts/to-do-edit.vue]: hi @harish86 , i think we can change Element -> ElementUI also here 
https://github.com/cerclecrm/cercle/blob/autosize-with-markdown/.eslintrc.js#L22

Comment (parallel588) [inline web/static/components/contacts/to-do-edit.vue]: I did it :)

Comment (harish86) [inline web/static/components/contacts/to-do-edit.vue]: @parallel588 thanks

**Pull request merged**